layout/js/map.js

```diff
@@ -157,10 +158,12 @@ function hovertext(text, evt){
     stateVal = " ";
   } else {
     var ref = evt.target.getAttribute('xlink:href').split('-')[0];
-    var stateName = ref.replace(/#/g, '')
+    var stateName = ref.replace(/#/g, '');
     var displayNum = Math.round(get_state_value(stateName));
     if (isNaN(displayNum)){
       displayNum = 'no data';
+    } else {
+      displayNum = displayNum.toLocaleString() + ' mgd';
```

Comment:
I think toLocaleString should be good, though it doesn't work if somehow a non-number sneaks in there.  Just something to watch out for.